src/parameters.toml

```diff
@@ -538,11 +538,23 @@ alias = "eccentricity_epoch"
 value = 0.016708634
 type = "float"
 
-[longitude_perihelion]
-alias = "lon_perihelion"
-value = 4.938188299449
+[orbit_obliquity_spline]
```

Comment:
I think changes in this file are the only ones really needed since we're moving to the new interface relatively soon